code/modules/antagonists/clock_cult/scriptures/summon_marauder.dm

```diff
@@ -0,0 +1,35 @@
+//==================================//
+// !           Cogscarab          ! //
+//==================================//
+/datum/clockcult/scripture/marauder
+	name = "Summon Clockwork Marauder"
+	desc = "Summons a Clockwork Marauder, a powerful warrior that can deflect ranged attacks."
+	tip = "Use Clockwork Marauders as an expendable soldier to send into combat when the fighting gets rough."
```

Comment:
if these are meant to be expendable, their power definitely needs some lookin at

Reply:
Adjective changed